defaultclient.go

```diff
@@ -242,3 +252,73 @@ func (client *DefaultClient) HealthCheck() bool {
 	}
 	return true
 }
+
+// ValidateAudienceScope validate audience and scope of user access token
+func (client *DefaultClient) ValidateAudienceScope(claims *JWTClaims, scope string) error {
+	if baseURI == "" {
+		err := client.getClientInformation(claims.Namespace)
+		if err != nil {
+			return err
+		}
+	}
+
+	if scope == "" {
+		scope = defaultScope
```

Comment:
this is wrong, setting default scope happens in IAM, not in the SDK. SDK should verify the scope as-is

Reply:
okay, i will remove it
thanks!